project-page.html

```diff
@@ -0,0 +1,395 @@
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/variables.css">
+    <link rel="stylesheet" href="styles/components.css">
+    <link rel="stylesheet" href="styles/project-page.css">
+</head>
+
+<body>
+    <div class="container">
+        <header class="header">
+            <h1 class="header__page-title">
+                PROJECT COMMANDS
+            </h1>
+        </header>
+        <div class="content">
+            <div class="sidebar">
+                <ul class="side-nav">
+                    <li class="side-nav__item">
+                        <a href="#" class="side-nav__link">
+                            <svg class="side-nav__icon">
+                                <use xlink:href="img/sprite.svg#icon-home"></use>
+                            </svg>
+                            <span>HOME</span>
+                        </a>
+                    </li>
+                    <li class="side-nav__item side-nav__item--active">
+                        <a href="#" class="side-nav__link">
+                            <svg class="side-nav__icon">
+                                <use xlink:href="img/sprite.svg#icon-aircraft-take-off"></use>
+                            </svg>
+                            <span>PROJECT</span>
+                        </a>
+                    </li>
+                    <li class="side-nav__item">
+                        <a href="#" class="side-nav__link">
+                            <svg class="side-nav__icon">
+                                <use xlink:href="img/sprite.svg#icon-key"></use>
+                            </svg>
+                            <span>CHANGES</span>
+                        </a>
+                    </li>
+                    <li class="side-nav__item">
+                        <a href="#" class="side-nav__link">
+                            <svg class="side-nav__icon">
+                                <use xlink:href="img/sprite.svg#icon-map"></use>
+                            </svg>
+                            <span>BRANCH</span>
+                        </a>
+                    </li>
+                    <li class="side-nav__item">
+                        <a href="#" class="side-nav__link">
+                            <svg class="side-nav__icon">
+                                <use xlink:href="img/sprite.svg#icon-map"></use>
+                            </svg>
+                            <span>ABOUT US</span>
+                        </a>
+                    </li>
+                </ul>
+                <div class="legal">
+                    &copy; 2024 by Git-Project
+                </div>
+            </div>
+            <section class="main-content">
+                    <article>
+                        <div class="article__item">
+                            <h3 class="sub-header">Details the relevant git commands relevant to creating a project locally.</h3>
```

Comment:
you can change the title "The relevant git commands to create a project locally"

Reply:
okay😜